release-please-config.json

```diff
@@ -1,20 +1,19 @@
 {
+  "release-type": "helm",
+  "skip-github-release": true,
+  "extra-files": [
+    {
+      "type": "yaml",
+      "path": "Chart.yaml",
+      "jsonpath": "$.version"
+    }
+  ],
   "packages": {
     "charts/platform": {
-      "component": "platform",
-      "changelog-path": "CHANGELOG.md",
```

Comment:
I removed settings that were configured to their defaults like `changelog-path, bump-minor-pre-major, bump-patch-for-minor-pre-major, draft, prerelease`.

The rest I moved to the top level so they are shared between components